### PR TITLE
protobuf 3.13.0

### PR DESCRIPTION
**Pull request by AustinShalit**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by AustinShalit** [comment]
This is ready for review.

File: Formula/monero.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Monero < Formula
       tag:      "v0.16.0.3",
       revision: "7bd1ed03dd29d1f194503b6fdce6588306328b72"
   license "BSD-3-Clause"
+  revision 2
```
Comment (SMillerDev) [inline Formula/monero.rb:8]:
```suggestion
  revision 1
```

**Review by SMillerDev — COMMENTED** [review]
Monero upstream isn't reachable from CI, could you try and reproduce it or report it upstream

**Comment by AustinShalit** [comment]
`brew audit monero --online --git --skip-style` passes with no errors locally. Just to check to make sure Actions could reach it, I ran a test: https://github.com/AustinShalit/actions-test/runs/996442640?check_suite_focus=true

How else can I see why audit thinks it is not reachable?

**Comment by gromgit** [comment]
It's a known certificate issue (#58854) that seems to be currently resolved by ignoring it (#59691, #59002, #58817, #58679).

**Comment by bayandin** [comment]
> It's a known certificate issue (#58854) that seems to be currently resolved by ignoring it (#59691, #59002, #58817, #58679).

Should be fixed by https://github.com/Homebrew/brew/pull/8412 for the future builds

**Comment by bayandin** [comment]
Thank you @AustinShalit, well done!

**Pull request closed without merging**